src/js/core/core.js

```diff
@@ -179,6 +186,18 @@
 			return null;
 		};
 
+		ns._TAUException = TAUException;
+
+		/**
+		 * Method throws TAU exception with given message
+		 * @param {string} message
+		 * @method throws
+		 * @member ns
+		 */
+		ns.throws = function (message) {
```

Comment:
How about  checking the type of message whether it is string or not?

Reply:
This is good idea. I'll add the checking.